internal/server/server.go

```diff
@@ -30,11 +30,19 @@ import (
 	"cloud.google.com/go/pubsub"
 	"cloud.google.com/go/storage"
 	"github.com/datacommonsorg/mixer/internal/base"
+	pb "github.com/datacommonsorg/mixer/internal/proto"
 	"github.com/datacommonsorg/mixer/internal/store"
 	"github.com/datacommonsorg/mixer/internal/translator"
 	"github.com/datacommonsorg/mixer/internal/util"
 )
 
+// Cache holds cached data for the mixer server.
+type Cache struct {
```

Comment:
comment on what each cache is about?

Reply:
How about naming with "ParentSvg", "SvgInfo" and "SvgSearchIndex" to make them consistent.

Reply:
done